roles/cs.varnish/templates/vcl/subroutines/recv.vcl.j2

```diff
@@ -131,7 +131,7 @@ if (
 ) {
     set req.http.{{ varnish_debug_request_info_header_name }} = "Token-Verified; Cache-Bypassed; Xid=" + req.xid + "";
 
-    return (pipe);
+    return (pass);
```

Comment:
Reversed a mistake. `(pipe)` caused all varnish logic to be bypassed and was not the best choice here.